test/expect.js

```diff
@@ -1129,11 +1129,16 @@ describe('expect', function () {
         lenFn  = function() { return heroes.length };
 
     expect(fn).to.change(obj, 'value');
+    expect(fn).to.change(obj, 'value').by(5);
+
     expect(sameFn).to.not.change(obj, 'value');
+    expect(sameFn).to.not.change(obj, 'value').by(1);
+
     expect(sameFn).to.not.change(obj, 'str');
     expect(bangFn).to.change(obj, 'str');
-    expect(batFn).to.change(lenFn);
-    expect(sameFn).to.not.change(lenFn);
+
+    expect(batFn).to.change(lenFn).by(1);
+    expect(sameFn).to.not.change(lenFn).by(1);
```

Comment:
I'd like to see some test cases confirming when the deltas end up being negative, and also some error cases.